Zend/tests/001.phpt

```diff
@@ -40,20 +40,23 @@ class test {
 }
 
 test::test1(1);
-var_dump(func_num_args());
+
+try {
+    func_num_args();
+} catch (Error $exception) {
+    echo $exception->getMessage() . "\n";
+}
 
 echo "Done\n";
 ?>
 --EXPECTF--
 int(0)
 int(1)
-Exception: Too few arguments to function test2(), 0 passed in %s001.php on line 18 and exactly 1 expected
+Exception: Too few arguments to function test2(), 0 passed in %s on line %d and exactly 1 expected
```

Comment:
It seems very uncommon for me that this exception message contains the file name and line too. Wouldn't it be better not to include it, just like what is done for internal calls?

As far as I saw, error monitoring tools like Sentry can't aggregate messages that have variations in the message. (P.S. I've just realized that it's another counterargument for displaying values in exception messages).

Reply:
This is because there are two different locations: The error message includes the caller location, while the error location is the callee location.